File: Javascript/ImplementStackUsingQueues.js
var Myqueue = function(){
    this.queue = []
}

Myqueue.prototype.push = function(x){
    this.queue.push(x)
}

Myqueue.prototype.pop = function(){
    for(let i=0;i<this.queue.length-1;i++){
        this.queue.push(this.queue.shift())
    }
    return this.queue.shift()
}

Myqueue.prototype.top = function(){
    return this.queue[this.queue.length-1]
}

Myqueue.prototype.empty = function(){
    if(this.queue.length == 0){
        return true
    }
    else{
        return false
    }
}

File: Python/ImplementStackUsingQueues.py
class Mystack:

    def __init__(self) -> None:
        self.stack = []


    def push(self,x):
        self.stack.push(x)

    def pop(self):
        self.stack.pop()

    def top(self):
        self.stack[-1]
    
    def empty(self):
        if len(self.stack) == 0:
            return True
        else:
            return False